indigo/build.sbt

```
@@ -1,40 +1,39 @@
-import scala.sys.process._
 import scala.language.postfixOps
-
-import sbtwelcome._
+import Misc._
 
 Global / onChangedBuildSource := ReloadOnSourceChanges
 
+val scala3Version = "3.1.0"
+
 ThisBuild / versionScheme                                  := Some("early-semver")
 ThisBuild / scalafixDependencies += "com.github.liancheng" %% "organize-imports" % "0.5.0"
+ThisBuild / scalaVersion := scala3Version
 
 lazy val indigoVersion = IndigoVersion.getVersion
 // For the docs site
 lazy val indigoDocsVersion  = "0.11.0"
 lazy val scalaJsDocsVersion = "1.8.0"
 lazy val scalaDocsVersion   = "3.1.0"
-//
-
-val scala3Version = "3.1.0"
 
 lazy val commonSettings: Seq[sbt.Def.Setting[_]] = Seq(
   version            := indigoVersion,
-  scalaVersion       := scala3Version,
   crossScalaVersions := Seq(scala3Version),
   organization       := "io.indigoengine",
-  libraryDependencies ++= Seq(
-    "org.scalameta" %%% "munit" % "0.7.29" % Test
-  ),
+  libraryDependencies ++= Dependencies.commonSettings.value,
   testFrameworks += new TestFramework("munit.Framework"),
   Test / scalaJSLinkerConfig ~= { _.withModuleKind(ModuleKind.CommonJSModule) },
   scalacOptions ++= Seq("-language:strictEquality"),
-  crossScalaVersions := Seq(scala3Version),
   scalafixOnCompile  := true,
   semanticdbEnabled  := true,
   semanticdbVersion  := scalafixSemanticdb.revision,
   autoAPIMappings    := true
```

SBT welcome shows an ugly `Example` logo for subprojects otherwise.

```suggestion
  autoAPIMappings    := true,
  logo               := name.value
```